tree/tree/inc/TTree.h

```diff
@@ -462,7 +462,36 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
    TDirectory             *GetDirectory() const { return fDirectory; }
    virtual Long64_t        GetEntries() const   { return fEntries; }
    virtual Long64_t        GetEntries(const char *selection);
-   virtual Long64_t        GetEntriesFast() const   { return fEntries; }
+   /// Return the number of entries in the dataset as currently known.
```

Comment:
Another way of stating could be "number greater or equal to the total number entries in the dataset".  "In addition we have `  LoadTree( ev ), ev < GetEntriesFast() ` that is guaranteed to be true only if  `ev less than the actual number of entries in the TChain` because when `ev` reach the last file in the TChain, `LoadTree` will update `GetEntriesFast` to return the actual number of entries.

Reply:
Thanks! I added your comment to the docs 👍